routes/articles.router.js

```diff
@@ -0,0 +1,16 @@
+const articlesRouter = require("express").Router();
+const {
+  getArticleById,
+  patchArticleById,
+  getArticles,
+  getCommentsById,
+  postCommentById,
+} = require("../controllers/articles.controllers");
+
+articlesRouter.get("/:article_id", getArticleById);
```

Comment:
If you want a further refactor, you can chain on methods: 

router.route('/').get(getFunction).patch(patchFunction).post(postFunction) etc. Give it a go!